### ISSUE TITLE
Non-zero status code returned while running Squeeze node

### ISSUE THREAD

**Issue by hoaquocphan** [onnx/tensorflow-onnx#1418]
Dear All,

I would like to report the issue and need the support from you.

1/ download tensorflow model from tensorflow hub: https://tfhub.dev/google/imagenet/mobilenet_v1_025_128/classification/4
2/ use below command to convert saved model to onnx model:
`python3 -m tf2onnx.convert --saved-model imagenet_mobilenet_v1_025_128_classification_4/ --output imagenet_mobilenet_v1_025_128_classification_4.onnx --opset 12 --concrete_function 2 --inputs inputs:0[1,224,224,3] --inputs-as-nchw inputs:0`

3/ I use the imagenet_mobilenet_v1_025_128_classification_4.onnx  as the input for other tool.
and the error log as below:
`onnxruntime.capi.onnxruntime_pybind11_state.RuntimeException: [ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Non-zero status code returned while running Squeeze node. Name:'StatefulPartitionedCall/StatefulPartitionedCall/predict/MobilenetV1/Logits/SpatialSqueeze' Status Message: /onnxruntime_src/onnxruntime/core/providers/common.h:18 int64_t onnxruntime::HandleNegativeAxis(int64_t, int64_t) axis >= -tensor_rank && axis <= tensor_rank - 1 was false. axis 2 is not in valid range [-2,1]`
As my checking, the error log is the format of tf2onnx tool. so I think that the issue because of step 2: convert from saved model to onnx. 

So could anyone know this issue or be familiar with this error log. Please explain for me and show me the solution?

**Comment by TomWildenhain-Microsoft** [comment]
> I use the imagenet_mobilenet_v1_025_128_classification_4.onnx as the input for other tool.

Which other tool?  The model is successfully converting and running for me.

**Comment by hoaquocphan** [comment]
Hi @TomWildenhain-Microsoft 

Thank for your reply!

I already check the onnx model and it is ok.
other tool is tool to convert onnx model to the other format, but it print the error log same as log format of tf2onnx tool so I think the error from the tf2onnx tool. 
so did you ever have this error log like that, such as: **HandleNegativeAxis**, **axis >= -tensor_rank && axis <= tensor_rank - 1**, **axis 2 is not in valid range**

**Comment by TomWildenhain-Microsoft** [comment]
The error message is saying that the input to the `SpatialSqueeze` op has a rank of 2, so axis 2 does not exist.  This doesn't make a lot of sense since the input to that node really ought to have a rank of 4.

The error is happening within onnx runtime.  I don't think I can help without knowing what tool you are using.

**Comment by hoaquocphan** [comment]
Hi @TomWildenhain-Microsoft 

Thank for your support

I will continues to investigate this issue as your comment and inform if there are any thing related to the tf2onnx tool.

**Comment by EmreOzkose** [comment]
Hello @hoaquocphan, did you find any solution about that? I have the same one.

**Comment by TomWildenhain-Microsoft** [comment]
@EmreOzkose are you getting your error in onnxruntime from a model converted with tf2onnx?

**Comment by hoaquocphan** [comment]
Hi @EmreOzkose 
this error log from my own tool, after update the tool, this issue is disappear.

**Comment by TomWildenhain-Microsoft** [comment]
@hoaquocphan glad to hear your issue was resolved.  I'm going to close this issue and @EmreOzkose you can open a new issue with your model if there is still a problem.  Please upload both your TF and onnx models if possible.
